inst/prepare_package_cran.sh

```diff
@@ -0,0 +1,24 @@
+echo "==> Switching to cran-release branch."
```

Comment:
You need to specify interpreter in the first line of the script! e.g. `#!/bin/bash`